sdk/translation/Azure.AI.Translation.Document/samples/Sample1_StartTranslation.md

```diff
@@ -3,7 +3,7 @@ This sample demonstrates how to translate one or more documents in a blob contai
 
 ## Creating a `DocumentTranslationClient`
 
-To create a new `DocumentTranslationClient` to run a translation operation for documents, you need a Translator endpoint and credentials. In the sample below, you'll use a Translator API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+To create a new `DocumentTranslationClient` to run a translation operation for documents, you need a Translator endpoint and credentials. You can use the [DefaultAzureCredential][DefaultAzureCredential] to try a number of common authentication methods optimized for both running as a service and development. In the sample below, you'll use a Translator API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
```

Comment:
Doesn't seem like there's a link for [DefaultAzureCredential]

Reply:
plop u are right. Thank you!
I added them to all samples.md files